kubeflow/common/ambassador.libsonnet

```diff
@@ -20,6 +20,10 @@
             name: "ambassador",
             port: 80,
             targetPort: 80,
+            [if (params.ambassadorServiceType == 'NodePort') && 
+                (params.ambassadorNodePort >= 30000) && 
+                (params.ambassadorNodePort <= 32767) 
+             then 'nodePort']: params.ambassadorNodePort,
```

Comment:
```: params.ambassadorNodePort,```should be within if condition as well?

Reply:
Apparently If the the if statement evaluates to false, libsonnet will not include the rest of the line in the yaml. I left the code this way and tested to make sure it behaves properly in the different scenarios.